### PR TITLE
Add ability to change e2e-benchmarking repo and branch

### PR DESCRIPTION
**Pull request by nathan-weinberg**
For additional flexibility, tested here: https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/scale-ci/job/nweinber-e2e-benchmarking-multibranch-pipeline/job/netobserv-perf-tests/538/

### CODE REVIEW

**Comment by nathan-weinberg** [comment]
cc @jotak @msherif1234 once this PR is merged all QE perf tests will use the lower `kafkaBatchSize` regardless of operator default (since we are overriding via FC)

**Comment by jotak** [comment]
@nathan-weinberg sounds good ... curious to see a ingress-perf run, it's the only one that we haven't checked for this afaik?

**Comment by memodi** [comment]
@nathan-weinberg - how about removing this option altogether so we test with whatever the default Operator configures? this is also in sync with our other theme to use defaults wherever possible unless necessary to use custom value

File: scripts/netobserv/flows_v1beta2_flowcollector.yaml [hunk 1]
```diff
@@ -11,7 +11,7 @@ spec:
       excludeInterfaces:
       - lo
       imagePullPolicy: IfNotPresent
-      kafkaBatchSize: 10485760
+      kafkaBatchSize: 1048576
```
Comment (memodi) [inline scripts/netobserv/flows_v1beta2_flowcollector.yaml:14]: @nathan-weinberg - how about removing this option altogether so we test with whatever the default Operator configures? this is also in sync with our other theme to use defaults wherever possible unless necessary to use custom value

**Comment by nathan-weinberg** [comment]
> @nathan-weinberg sounds good ... curious to see a ingress-perf run, it's the only one that we haven't checked for this afaik?

I'll be doing runs for all the workloads with the 1.5 GA image

File: scripts/netobserv/flows_v1beta2_flowcollector.yaml (hunk 1 above)
Comment (nathan-weinberg) [reply scripts/netobserv/flows_v1beta2_flowcollector.yaml:14]: yeah, i'm open to that

**Pull request merged**